src/features/quality-stats/StatsAggregator.js

```diff
@@ -75,6 +78,13 @@ class StatsAggregator {
         stats.packetsLostVariance = standardizedMoment(
             this._calculateSeriesDifferences(packetsLost), 2);
 
+        if (samplesReceived.length && concealedSamples.length) {
+            const concealed = concealedSamples.at(-1);
```

Comment:
Is concealedSamples reported cumulatively like totalSamplesReceived? because I noticed that the naming convention from the report seems to be different one is `totalSamplesReceived` and the other is simply `concealed`.